src/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -345,12 +345,7 @@ class UpdateUserPublicKey(_AnsiblePlaybookAlias):
     )
 
     def run(self, args, unknown_args):
-        args.playbook = 'deploy_stack.yml'
-        unknown_args += (
-            '--tags=pubkey',
-            '--extra-vars={{"dev_users": {{"present": [{}]}}}}'.format(args.username),
-        )
-        return AnsiblePlaybook(self.parser).run(args, unknown_args)
+        return "update-user-key command is deprecated. Use update-users command."
```

Comment:
I think the return value is supposed to be the process return code (like 0 for success, 1 or some other number for error). I think you can have it print this message. Here's an example of a similar place we do this https://github.com/dimagi/commcare-cloud/blob/47c5f6ee16c850996c48f438dd42b8e760786d7b/src/commcare_cloud/commands/ansible/run_module.py#L175-L177 (this will print it as a yellow "notice").

Reply:
Noted Danny. Will update in my next commit. thank you.